### ISSUE TITLE
Cluster global / Rancher global certificates / registries

### ISSUE THREAD

**Issue by hameno** [rancher/rancher#13331]
**Rancher versions:**
rancher/server or rancher/rancher: 2.0.0

It would be nice to be able to provision global shared certificates and registries for all projects / clusters.

**Comment by jannylund** [comment]
I agree. This is becoming a major pain when you have multiple projects but the same internal repository. (Combined with that k8s does not re-read registrys if you already createad a deployment, so you need to wipe the deployment after a registry was added).

**Comment by at24106** [comment]
The handling of rancher of missing global secretes is a pain, please can you build in this feature

**Comment by bradmesserle** [comment]
why is this still open? maybe close it and use trust-manager?
